### PR TITLE
Add Prerequisite to install Java OpenJDK8

### PR DESCRIPTION
**Pull request by daniarherikurniawan**
When following this [Readme](https://github.com/Seagate/cortx-s3server/blob/main/docs/CORTX-S3%20Server%20Quick%20Start%20Guide.md), we will get an error when running `./init.sh -a`. The error is as follows:
```sh
... as a requirement of jboss-servlet-3.0-api.noarch 0:1.0.1-9.el7 - u", "looking for ('java', None, (None, None, None)) as a requirement of jboss-servlet-3.0-api.noarch 0:1.0.1-9.el7 - u", "--> Finished Dependency Resolution", "Dependency Process ending", "Depsolve time: 0.802", "Error: Package: 1:java-1.8.0-openjdk-1.8.0.262.b10-0.el7_8.x86_64 (C7.8.2003-updates)", "           Requires: java-1.8.0-openjdk-headless(x86-64) = 1:1.8.0.262.b10-0.el7_8", "           Installed: 1:java-1.8.0-openjdk-headless-1.8.0.292.b10-1.el7_9.x86_64 (@updates)", "               java-1.8.0-openjdk-headless(x86-64) = 1:1.8.0.292.b10-1.el7_9", "           Available: 1:java-1.8.0-openjdk-headless-1.8.0.242.b08-1.el7.x86_64 (C7.8.2003-base)", "               java-1.8.0-openjdk-headless(x86-64) = 1:1.8.0.242.b08-1.el7", "           Available: 1:java-1.8.0-openjdk-headless-1.8.0.252.b09-2.el7_8.x86_64 (C7.8.2003-updates)", "               java-1.8.0-openjdk-headless(x86-64) = 1:1.8.0.252.b09-2.el7_8", "           Available: 1:java-1.8.0-openjdk-headless-1.8.0.262.b10-0.el7_8.x86_64 (C7.8.2003-updates)", "               java-1.8.0-openjdk-headless(x86-64) = 1:1.8.0.262.b10-0.el7_8", " You could try using --skip-broken to work around the problem", " You could try running: rpm -Va --nofiles --nodigest"]}
```

This problem has been confirm by @mengwanguc as well. We tested this on Chameleon Servers. Let me know if you guys prefer another form of solution. Thanks :)

-----
[View rendered docs/CORTX-S3 Server Quick Start Guide.md](https://github.com/daniarherikurniawan/cortx-s3server/blob/patch-2/docs/CORTX-S3 Server Quick Start Guide.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: docs/CORTX-S3 Server Quick Start Guide.md [hunk 1]
```diff
@@ -77,9 +77,14 @@ This guide provides a step-by-step walkthrough for getting you CORTX-S3 Server r
      Once you power on your VM, you can verify if selinux and firewall are disabled by using: `$ getenforce` - you'll get a 'disabled' status.
 
 
-9. You'll need to install CORTX Python Utilities. Follow the steps to install [CORTX Python Utilities](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md).
+9. You'll need to install Java OpenJDK 8 by running
```
Comment (nileshgovande) [inline docs/CORTX-S3 Server Quick Start Guide.md:80]: pls change the wording to 'if not already present'

Comment (nileshgovande) [reply docs/CORTX-S3 Server Quick Start Guide.md:80]: also pls provise the command 'rpm -qa | grep java' so that one can check if it exists

Comment (daniarherikurniawan) [reply docs/CORTX-S3 Server Quick Start Guide.md]: Hi @nileshgovande , I updated the PR according to your comments. BTW, the `yum install *` will by default checks whether the package exists or not; it won't install anything if the package is already installed or satisfied.
Let me know if I should do any other changes. Thx!

**Review by nileshgovande — APPROVED** [review]

**Pull request merged**